package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -1029,6 +1102,13 @@ def _write_next_frame(self, ag):
 
         return self._write_next_timestep(ts)
 
+    @staticmethod
+    def _scale(variable, scale_factor):
+        if scale_factor is None or scale_factor == 1:
+            return variable
+        else:
+            return variable / scale_factor
+
```

Comment:
Due to `mmap` I _think_ reassigning `value` is problematic. I can look into doing the assignment in the function it really makes things clearer

Reply:
Does assigning `values` to a different variable name inside the function get around that?

Reply:
I can't remember the exact details of `mmap` but iirc the TLDR was "the least amount of things that reference the underlying python object the better".

Can just add an extra line and not having to worry about it though 🙃

Reply:
Sorry my bad, I messed up, this is write not read (probably should have named the functions differently...).

Modifying `value` will modify things like the timestep positions inplace (if self.convert_units is False), it's not an issue since it'll get rewritten once you move away from that timestep, but the original intent was to avoid doing any inplace modification if possible (see `convert_units` stuff).

Anyways, still doable, just needs that extra `else` 🙃